test/Microsoft.DotNet.Tools.Tests.Utilities/UnixOnlyFactAttribute.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.PlatformAbstractions;
+using Xunit;
+
+namespace Microsoft.DotNet.Tools.Test.Utilities
+{
+    public class UnixOnlyFactAttribute : FactAttribute
```

Comment:
Is this left over from the other changes? Something that we will use again when we add the new properties back?

Reply:
I figured it was generally useful - you'll need it sooner or later. I won't port it to preview.